### PR TITLE
Academic sponsor group

### PR DESCRIPTION
**Pull request by JaswiniP**
update couple of locators

### CODE REVIEW

File: Web/Edubase.Web.UI/Assets/Scripts/Entry/merge-amalgamate.js [hunk 1]
```diff
@@ -425,7 +425,7 @@ const mergersApp = new Vue({
 
       if (this.mergeDateError) {
         this.errors.push({
-          href: '#amalgamationdate-day',
+          href: '#amalgamation-date-day',
```
Comment (antk43) [inline Web/Edubase.Web.UI/Assets/Scripts/Entry/merge-amalgamate.js:428]: @JaswiniP should the value be changed here as we are only changing ids? FYI @RogerHowellDfE

File: Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml [hunk 2]
```diff
@@ -260,8 +260,8 @@
                         <div class="govuk-date-input">
                             <div class="govuk-date-input__item">
                                 <div class="govuk-form-group">
-                                    <label for="amalgamationdate-day" class="govuk-label govuk-date-input__label">Day</label>
-                                    <input class="govuk-input date-text-day" type="text" v-model="mergeDateDay" id="amalgamationdate-day">
+                                    <label for="amalgamation-date-day" class="govuk-label govuk-date-input__label">Day</label>
```
Comment (antk43) [inline Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml:263]: @JaswiniP should the for be changed here as we are only changing ids? FYI @RogerHowellDfE

Comment (RogerHowellDfE) [reply Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml:263]: The `for` attribute of a `<label>` needs to match the `id` value of the `<input>` element the label is describing - updating one needs the other to be updated also, so updating both here is fine :)

File: Web/Edubase.Web.UI/Assets/Scripts/Entry/merge-amalgamate.js (hunk 1 above)
Comment (RogerHowellDfE) [reply Web/Edubase.Web.UI/Assets/Scripts/Entry/merge-amalgamate.js:428]: An `href` with an `#` in it is (usually) a link to the HTML element with that `id` on the page (similar to CSS selectors). 

If updating the ID, it makes sense to also update links which point to it so looks good here :)

**Pull request closed without merging**